app/build.gradle

```diff
@@ -137,15 +137,15 @@ dependencies {
     testImplementation "junit:junit:${junitVer}"
     testImplementation "org.mockito:mockito-all:${mockitoVer}"
     testImplementation "com.squareup.assertj:assertj-android:${androidAssertJVer}"
-    testImplementation("org.robolectric:robolectric:3.5.1") {
+    testImplementation('org.robolectric:robolectric:3.6.1') {
         exclude group: 'commons-logging', module: 'commons-logging'
         exclude group: 'org.apache.httpcomponents', module: 'httpclient'
     }
-    testImplementation("org.robolectric:shadows-support-v4:3.3.2") {
+    testImplementation('org.robolectric:shadows-supportv4:3.6.1') {
         exclude group: 'commons-logging', module: 'commons-logging'
         exclude group: 'org.apache.httpcomponents', module: 'httpclient'
     }
-    testImplementation 'io.reactivex:rxjava:1.3.4'
+    testImplementation 'io.reactivex.rxjava2:rxjava:2.1.0'
```

Comment:
same as above

Reply:
I'm not doing it right now, I will create ticket to clean up `build.gradlez